tools/misc/browse_dataset.py

```diff
@@ -217,11 +244,10 @@ def main():
             show_det_data(input, args.output_dir, show=args.online)
         if vis_task in ['multi_modality-det', 'mono-det']:
             # project 3D bboxes to 2D image
-            show_proj_bbox_img(
```

Comment:
after change this function name, public dataset such as kitti or nuscenes will not be used normally.
may be you can add a new task name or adapt the raw interface?